### ISSUE TITLE
building a new .exe does not work

### ISSUE THREAD

**Issue by fushino009** [Jire/Charlatano#822]
anyone with a good heart wants to share the latest build for charlatano i cant seem to make mine work even if i build a new one it fails thank you in advance

**Comment by wajabirap** [comment]
If I was you I would post the screenshot of the error so someone can try helping you. It'll help you compile future builds instead of waiting for someone to help. I would post the build with you but it's 54.6 MB and GitHub only allows file sizes of 10MB.

**Comment by fushino009** [comment]
> If I was you I would post the screenshot of the error so someone can try helping you. It'll help you compile future builds instead of waiting for someone to help. I would post the build with you but it's 54.6 MB and GitHub only allows file sizes of 10MB.

yeah how about mediafire? can you up it in media fire or google drive? ill say thank you in advance :)

**Comment by wajabirap** [comment]
> > If I was you I would post the screenshot of the error so someone can try helping you. It'll help you compile future builds instead of waiting for someone to help. I would post the build with you but it's 54.6 MB and GitHub only allows file sizes of 10MB.
> 
> yeah how about mediafire? can you up it in media fire or google drive? ill say thank you in advance :)

I could have modified/infected the file. You don't really know so be cautious when dealing with files you get from random people from the internet. I also tested it in-game and it works but all the settings are the default values.
Here's the link to the file - https://www.mediafire.com/file/xhoaxy5dpyb109u/Charlatano_1.0.10.zip/file

**Comment by fushino009** [comment]
> > > If I was you I would post the screenshot of the error so someone can try helping you. It'll help you compile future builds instead of waiting for someone to help. I would post the build with you but it's 54.6 MB and GitHub only allows file sizes of 10MB.
> > 
> > 
> > yeah how about mediafire? can you up it in media fire or google drive? ill say thank you in advance :)
> 
> I could have modified/infected the file. You don't really know so be cautious when dealing with files you get from random people from the internet. I also tested it in-game and it works but all the settings are the default values.
> Here's the link to the file - https://www.mediafire.com/file/xhoaxy5dpyb109u/Charlatano_1.0.10.zip/file

thank you my good sir :)

**Comment by karmalegend** [comment]
What error where you getting tho?

**Comment by Hiseaks** [comment]
the wall is not run @@ :(((((((

**Comment by pravstf97** [comment]
> What error where you getting tho?

The walls are still broke when the enemy is behind walls cant see but when he is in frotn of me he is in red colour...Any solution ?

**Comment by pravstf97** [comment]
> > > If I was you I would post the screenshot of the error so someone can try helping you. It'll help you compile future builds instead of waiting for someone to help. I would post the build with you but it's 54.6 MB and GitHub only allows file sizes of 10MB.
> > 
> > 
> > yeah how about mediafire? can you up it in media fire or google drive? ill say thank you in advance :)
> 
> I could have modified/infected the file. You don't really know so be cautious when dealing with files you get from random people from the internet. I also tested it in-game and it works but all the settings are the default values.
> Here's the link to the file - https://www.mediafire.com/file/xhoaxy5dpyb109u/Charlatano_1.0.10.zip/file

The walls are still broke when the enemy is behind walls cant see but when he is in frotn of me he is in red colour...Any solution ?

**Comment by wajabirap** [comment]
> > > > If I was you I would post the screenshot of the error so someone can try helping you. It'll help you compile future builds instead of waiting for someone to help. I would post the build with you but it's 54.6 MB and GitHub only allows file sizes of 10MB.
> > > 
> > > 
> > > yeah how about mediafire? can you up it in media fire or google drive? ill say thank you in advance :)
> > 
> > 
> > I could have modified/infected the file. You don't really know so be cautious when dealing with files you get from random people from the internet. I also tested it in-game and it works but all the settings are the default values.
> > Here's the link to the file - https://www.mediafire.com/file/xhoaxy5dpyb109u/Charlatano_1.0.10.zip/file
> 
> The walls are still broke when the enemy is behind walls cant see but when he is in frotn of me he is in red colour...Any solution ?

In the ESP.kts file change the value of MODEL_ESP to false and see if that fixes any of the issues.

**Comment by muhammadnasyreeq** [comment]
**Hi, is there any update on this hack?**
